loader/src/packfile_io.rs

```diff
@@ -82,7 +82,7 @@ impl PackfileReader {
             reader: message_reader,
             index_by_uuid,
             assets_by_path,
-            runtime: tokio::runtime::Builder::new_multi_thread().build()?,
+            runtime: bevy_tasks::IoTaskPool(bevy_tasks::TaskPoolBuilder::default().build()),
```

Comment:
Long-term we should probably think about how we can use a user-provided task pool